changelogs/current.yaml

```diff
@@ -15,3 +15,6 @@ removed_config_or_runtime:
 new_features:
 
 deprecated:
+- area: router
```

Comment:
I think this goes under removed config or runtime

Reply:
good catch, done.